src/verifiers/openAttestationDnsTxt.ts

```diff
@@ -73,24 +75,28 @@ export const openAttestationDnsTxt: Verifier<
       if (isWrappedV2Document(document)) {
         const documentData = getData(document);
         const identities = await Promise.all(
-          // we expect the test function to prevent this issue => smart contract address MUST be populated
-          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-          documentData.issuers.map(issuer => resolveIssuerIdentity(issuer, getSmartContractAddress(issuer)!, options))
+          documentData.issuers.map(issuer => {
```

Comment:
what happens if more than 1 issuer fails?

Reply:
we are all doomed :see_no_evil:

Reply:
Created issue to handle that case https://github.com/Open-Attestation/oa-verify/issues/78